src/renderer/src/components/song/song-detail/styles.css

```diff
@@ -1,5 +1,6 @@
 .song-detail {
   --padding: 52px;
+  --dynamic-color: black; /* Default dynamic color */
```

Comment:
Black is probably not the best choice for default color considering that the application is only in dark mode (yet). Ask @duduBTW what color to use

Reply:
Just keep as white for now.

Reply:
Considering we will also support light color themes, wouldn't something like a lighter gray be a better option atleast for future?

Reply:
The light mode is a whole nother beast, we will want to make it dynamic at that point:

- Black for light mode.
- White for dark mode.

Since we want only have the dark mode you can keep as a static white for now.